README.md

```diff
@@ -1 +1,44 @@
-The documentation is on [spilo.readthedocs.org](http://spilo.readthedocs.org/).
+## Spilo: HA PostgreSQL Clusters with Docker
+
+Spilo is a Docker image that provides a resilient, High Available PostgreSQL cluster (HA-cluster) that you can configure and start within minutes. Spilo can run several clusters at one time, with one cluster acting as a master and the others as slaves. Its name derives from სპილო [spiːlɒ], the Georgian word for "elephant." 
+
+Spilo is currently evolving: Its creators are working on a Postgres operator that would make it simpler to deploy scalable Postgres clusters in a Kubernetes environment, and do maintenance tasks. Spilo would serve as an essential building block for this.
+
+Spilo works in combination with [Patroni](https://github.com/zalando/patroni), a template for PostgreSQL HA. Patroni works with [etcd](https://coreos.com/etcd/), [ZooKeeper](https://zookeeper.apache.org/) or [Consul](https://www.consul.io/), and manages the [PostgreSQL](https://www.postgresql.org) databases running in your Spilo cluster. It ensures that there is only one master within your Spilo. (It's worth noting here that Patroni includes a Helm Chart that enables you to deploy a five-node Patroni cluster using a Kubernetes PetSet, in a Google Compute Engine environment.)
+
+###How to Use This Docker Image
+
+First, there are some technical requirements:
+- A Distributed Configuration Store (DCS), to allow distributed nodes to agree. Most Patroni deployments use etcd, which implements the [Raft protocol](https://github.com/coreos/etcd/tree/master/raft). For more information about Raft, visit [this resource](http://thesecretlivesofdata.com/raft/) and check out [this interactive visualization](https://raft.github.io).
+
+>>>**not sure if this is correct?** This image includes 5432 (the Postgres port). The default Postgres user and database are created in the entrypoint with initdb.
```

Comment:
This sentence belongs to the administrative connections section. Instead of the first sentence, It should say that PostgreSQL is configured by default to listen to port 5432